package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-appointment-slots-pickers",
-  "version": "1.0.0",
+  "version": "0.2.7",
```

Comment:
@szilvihorvath version number to `2.0.0` pls

Reply:
@somasuna thanks, updated